### ISSUE TITLE
[BUG] White screen when opening the installer

### ISSUE THREAD

**Issue by Crazyman-byte** [bloxstraplabs/bloxstrap#3380]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.0

### What problem did you encounter?

When ever I try opening the bloxstrap installer it always pops up as a white screen then goes to (not responding) Not sure if the log below is helpful I'll also attach a vid of it happening 

https://github.com/user-attachments/assets/31eb0e90-0708-4954-b7ef-a364d4f22e7f

note after a thing called bloxstrap menu opens up but its the same white screen but it only sometimes happens
here is the log the other text box wasn't working 
[UE4SS.log](https://github.com/user-attachments/files/17455024/UE4SS.log)

**Comment by pizzaboxer** [comment]
Try reinstalling the .NET 6 runtime. Find it by searching for "desktop runtime" in the add/remove programs list, and then reinstall it at https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.33-windows-x64-installer?cid=getdotnetcore

**Comment by Crazyman-byte** [comment]
Still doesn't work.

**Comment by totallyrandom001** [comment]
this may be caused by your hardware being too slow to load the application. i advise waiting for about 5 minutes since i've personally experienced this on old pc's running on hard disks. hope that helps!

**Comment by pizzaboxer** [comment]
> here is the log the other text box wasn't working
> [UE4SS.log](https://github.com/user-attachments/files/17455024/UE4SS.log)

Where did this log come from and what makes you think it's relevant to what's happening here?

**Comment by Crazyman-byte** [comment]
> this may be caused by your hardware being too slow to load the application. i advise waiting for about 5 minutes since i've personally experienced this on old pc's running on hard disks. hope that helps!

Yeah my pc is pretty new but I still tried your suggestion and it didn't work out. Thanks for trying to help your time is greatly appreciated

**Comment by pizzaboxer** [comment]
Looks like this log is relevant to the issue. Do you know what UE4SS is and do you know how to not make it apply to Bloxstrap?

**Comment by Crazyman-byte** [comment]
No clue I'll try doing some research.

**Comment by Crazyman-byte** [comment]
I didn't mean to close it mb
